core/arch/arm/plat-ls/ls_hw_unq_key_a64.S

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018 NXP
+ * All rights reserved.
```

Comment:
It is preferred **not** to add "All rights reserved", nor the full license text, unless you have a good reason to do so (such as your legal team absolutely requiring it). In other words, would you mind making the header be as follows:

```C
/* SPDX-License-Identifier: BSD-2-Clause */
/*
 * Copyright 2018 NXP
 */

#include <asm.S>
...
```

?

Reply:
Need some time to revert back on this comment.

Reply:
Accepted.